.github/workflows/lint-info-xml.yml

```diff
@@ -7,7 +7,6 @@ name: Lint info.xml
 
 on:
   pull_request:
-  push:
     branches:
```

Comment:
This was part of push and is now part of pr, intended?
Doesn't allow sending follow up prs, but fine either way for normal work